### ISSUE TITLE
Replace "Wrote screenshots/shot001.jpg" with an audible acknowledgement

### ISSUE THREAD

**Issue by wtfbbqhax** [wtfbbqhax/tremulous#10]
Too much damn nonsense is plumbed and logged directly to the console and screen. I DO NOT WANT A SCREENSHOT TO ADD SHIT TO THE SCREEN!!!

So dumb!

![24812671536_e508d3ea0a_z](https://cloud.githubusercontent.com/assets/7323455/17646535/16323a6a-619c-11e6-9947-c82646e79e5e.jpg)

---
## Solution [🔊 Click for example](https://www.freesound.org/people/lebcraftlp/sounds/242429/)

Stop writing a log message for screenshots. Instead play a comical classic camera "click" audio clip.

**Comment by wtfbbqhax** [comment]
### [🔊 Nikon D50 sounds](http://www.soundsnap.com/tags/d50)

I like these better; and License lgtm.

**Comment by wtfbbqhax** [comment]
Sound snap has started watermarking their in-browser sounds. Not sure about the extent.

**Comment by wtfbbqhax** [comment]
Closing this, I'll make the screenshot text "[skipnotify]" or something. Perhaps I'll add a Hud thing specifically that hides when taking the screenshot. 

As it is, there is currently no way to add a sound asset to the client binary without breaking compatibility with a server which hosts old 1.1 or gpp assets. If the "sound" doesn't exists in the pure paks loaded, the client will "Crash".. 

This is probably more useful as a separate issue, and is something I noticed when originally attempting to replace the default "Clank!" sound with "silence.wav".
